fasta/pFasta/pFasta_test.go

```diff
@@ -1,9 +1,12 @@
 package pFasta
 
 import (
-	"github.com/vertgenlab/gonomics/dna/pDna"
 	"math/rand"
+	"os"
```

Comment:
Thank you for catching this! For everyone else reviewing, this was a small bug in pFasta that Nikitha and I noticed in our meeting. Now fixed